SELinux: Build failure with `gardenlinux-selinux-module` and `selinux-policy-default `

**What happened:**

build fails with 

```
Created symlink /etc/systemd/system/cloud-init.target.wants/cloud-config.service -> /lib/systemd/system/cloud-config.service.
Created symlink /etc/systemd/system/cloud-init.target.wants/cloud-final.service -> /lib/systemd/system/cloud-final.service.
Created symlink /etc/systemd/system/cloud-init.target.wants/cloud-init-hotplugd.socket -> /lib/systemd/system/cloud-init-hotplugd.socket.
Created symlink /etc/systemd/system/cloud-init.target.wants/cloud-init-local.service -> /lib/systemd/system/cloud-init-local.service.
Created symlink /etc/systemd/system/cloud-init.target.wants/cloud-init.service -> /lib/systemd/system/cloud-init.service.
Processing triggers for libc-bin (2.34-3) ...
Processing triggers for dracut (056-3) ...
Errors were encountered while processing:
 gardenlinux-selinux-module
E: Sub-process /usr/bin/dpkg returned an error code (1)
make: *** [Makefile:73: aws-dev] Error 100
```

**Debug Logs:**

Fails on this commit

```
commit 68c84e15f59118f6581570bf1b8d639b787970a0 (HEAD -> main, origin/main, origin/HEAD)
```

**How to reproduce it**:

run `make aws-dev` on above commit.

 
**Environment:**

Fails on the garden linux dev machine 3.68.232.241

--- a/features/server/pkg.include
+++ b/features/server/pkg.include
@@ -32,5 +32,5 @@ quota
 nvme-cli
 selinux-basics
 selinux-policy-default
-gardenlinux-selinux-module
+#gardenlinux-selinux-module
 sosreport

